### PR TITLE
various formulae: undeprecate & update comments

### PR DESCRIPTION
**Pull request by p-linnane**
Per discussion in https://github.com/Homebrew/homebrew-core/pull/130367, this PR undeprecates some formulae that were deprecated due to their age. This PR also clarifies some deprecation comments and updates their reasons.

### CODE REVIEW

File: Formula/cronolog.rb [hunk 1]
```diff
@@ -22,10 +22,6 @@ class Cronolog < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "5b7e0546314cd92d44b009de1894f4739f75865da5f95d425e305cf6ffeac10d"
   end
 
-  # Upstream is gone.
```
Comment (SMillerDev) [inline Formula/cronolog.rb:25]: This is a reason for deprecation

File: Formula/chcase.rb [hunk 2]
```diff
@@ -9,10 +9,6 @@ class Chcase < Formula
     sha256 cellar: :any_skip_relocation, all: "c1ac32a52434724c8df47fe8ba642f69f58b55c8df2dd3a5a8bc62b4cdb07758"
   end
 
-  # Homepage is gone. No meaningful updates since the
```
[the remark this answers is not in the document]
Comment (MikeMcQuaid) [reply Formula/chcase.rb:12]: If it still works and analytics show it's still used: let's not unless we have a new deprecation state that doesn't get disabled/removed.

File: Formula/cronolog.rb (hunk 1 above)
Comment (MikeMcQuaid) [reply Formula/cronolog.rb:25]: If it still works and analytics show it's still used: let's not unless we have a new deprecation state that doesn't get disabled/removed.

**Review by MikeMcQuaid — APPROVED** [review]
Thanks!

File: Formula/ntp.rb [hunk 3]
```diff
@@ -19,6 +19,8 @@ class Ntp < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "16137579ea8978ef71a1222071cd9e44a42eeeaa8d83ed83bdbf0fd048e6178c"
   end
 
+  # Does not build with `openssl@3`
```
Comment (carlocab) [inline Formula/ntp.rb:22]: Arch build it fine with OpenSSL 3.0.8 though, so this we might be doing something wrong here

**Pull request merged**